apps/frontend/src/app/5_pages/AavePage/components/LendAssetsList/components/LendForm/LendForm.tsx

```diff
@@ -66,20 +62,19 @@ export const LendForm: FC<LendFormProps> = ({
     [reserves],
   );
 
-  const isValidLendAmount = useMemo(
-    () => (lendSize.gt(0) ? lendSize.lte(lendAssetBalance) : true),
-    [lendSize, lendAssetBalance],
-  );
-
-  const submitButtonDisabled = useMemo(
-    () => !isValidLendAmount || lendSize.lte(0),
-    [isValidLendAmount, lendSize],
-  );
+  const reserve = useMemo(() => {
+    return reserves.find(r => r.symbol === lendAsset) ?? reserves[0];
+  }, [reserves, lendAsset]);
```

Comment:
it crashes if reserves array is empty.

Reply:
Should be unreachable if reserves are empty, but I can keep it as undefined and handle it else where